cruise.umple/src/Generator_CodeAnalysis.ump

```diff
@@ -494,9 +494,9 @@ class McCabeMetricVisitor {
 	
 	// Trace
 	void visit(AttributeTraceItem attributeT) {
-		if(attributeT.isConditionallyWhere()) {
-			cur_conditions++;
-		}
+//		if(attributeT.isConditionallyWhere()) {

```

Comment:
Can you update the editor to have 2 spaces instead of tabs,  similar comment for below